code/game/objects/items/weapons/grenades/grenade.dm

```diff
@@ -86,7 +86,7 @@
 		return
 
 	switch(det_time)
-		if(1 DECISECONDS)
+		if(0.01 SECONDS)
```

Comment:
0.01 seconds is not a decisecond.
Also, `DECISECONDS` is a fine define to use

Reply:
I hate deciseconds I hate deciseconds why couldn't it have been miliseconds